theme.js

```diff
@@ -1 +1,36 @@
-import './js/blogtini.js'
+/**
+ * Dress up all the things
+ */
+
+/**
+ * What is the intented URL to expose this site, it's most likely over http, not "file://",
+ * and possibly not at the root of the domain name.
+ */
+const PRODUCTION_SITE_ROOT_BASE_URL = 'https://renoirb.github.io/blogtini/'
+
+/**
+ * This should work for file:/// URLs
+ * (assuming it's supported, Safari does, not Chromium thus far 2023-02-03)
+ */
+const SITE_ROOT_BASE_URL = await import.meta.url?.replace('theme.js', '') // await import.meta.resolve('./theme.js', import.meta.url) // ??'').replace('theme.js', '');
```

Comment:
Declare what's the current URL thanks to `import.meta.url`, if it's not the same as `PRODUCTION_SITE_ROOT_BASE_URL` it must be development.